.github/workflows/CD.yml

```diff
@@ -46,17 +46,21 @@ jobs:
       uses: appleboy/ssh-action@master
       env:
         SITE_URL: ${{ env.MASINDI_SITE_URL }}
+        MASINDI_TAG_ID: "G-JEX2JKPWJT"
+        MASINDI_DB_CONNECTION_STRING: ${{ secrets.MASINDI_DB_CONNECTION_STRING }}
       with:
         host: ${{ secrets.MASINDI_HOST }}
         username: ${{ secrets.MASINDI_USERNAME }}
         key: ${{ secrets.MASINDI_KEY }}
         port: ${{ secrets.MASINDI_PORT }}
         passphrase: ${{ secrets.MASINDI_PASSPHRASE }}
         command_timeout: 30m
-        envs: SITE_URL
+        envs: SITE_URL,MASINDI_TAG_ID,MASINDI_DB_CONNECTION_STRING
         script: |
           cd ${{ env.REPO_PATH }}
           echo "SITE_URL=${{ env.MASINDI_SITE_URL }}" > .env.nginx
+          echo "MASINDI_TAG_ID=${{ env.MASINDI_TAG_ID }}" >> .env
```

Comment:
Use one `>` for the first write to the `.env` file. Otherwise we shall end up with the file having multiple entries for the save variable for every deployment.

Reply:
Oh yes!